dali/operators/reader/video_reader_op.cc

```diff
@@ -26,63 +26,88 @@ namespace dali {
+      R"code(If the`` file_list`` or ``file_root`` arguments are passed, returns
+the timestamps output. )code",
       false)
   .AddArg("sequence_length",
       R"code(Frames to load per sequence.)code",
       DALI_INT32)
   .AddOptionalArg("step",
-      R"code(Frame interval between each sequence (if `step` < 0, `step` is set to `sequence_length`).)code",
+      R"code(Frame interval between each sequence.
+
+When the value is less than 0, ``step`` is set to ``sequence_length``.)code",
       -1)
   .AddOptionalArg("channels",
       R"code(Number of channels.)code",
       3)
   .AddOptionalArg("additional_decode_surfaces",
       R"code(Additional decode surfaces to use beyond minimum required.
-This is ignored when decoder is not able to determine minimum
-number of decode surfaces, which may happen when using an older driver.
-This parameter can be used trade off memory usage with performance.)code",
+
+This argument is ignored when the decoder cannot determine the minimum number of
+decode surfaces
+
+.. note::
+
+  This can happen when the driver is an older version.
+
+This parameter can be used to trade off memory usage with performance.)code",
       2)
   .AddOptionalArg("normalized",
-      R"code(Get output as normalized data.)code",
+      R"code(Gets the output as normalized data.)code",
       false)
   .AddOptionalArg("image_type",
-      R"code(The color space of the output frames (supports RGB and YCbCr).)code",
+      R"(The color space of the output frames (RGB or YCbCr).)",
       DALI_RGB)
   .AddOptionalArg("dtype",
-      R"code(The data type of the output frames (supports FLOAT and UINT8).)code",
+      R"(The data type of the output frames (UINT8 or FLOAT).)",
       DALI_UINT8)
   .AddOptionalArg("stride",
-      R"code(Distance between consecutive frames in sequence.)code", 1u, false)
+      R"code(Distance between consecutive frames in the sequence.)code", 1u, false)
   .AddOptionalArg("skip_vfr_check",
-      R"code(Skips check for variable frame rate on videos. This is useful when heuristic fails.)code", false)
+      R"code(Skips the check for the variable frame rate (VFR) videos.
+
+Use this flag to suppress false positive detection of VFR videos.
+
+.. warning::
+
+  When the dataset indeed contains VFR files, setting this flag may cause the decoder to
+  malfunction.)code", false)
   .AddOptionalArg("file_list_frame_num",
-      R"code(If start/end timestamps are provided in file_list, interpret them as frame
-numbers instead of timestamp. If floating point values are given, then
-start frame number is ceiling of the number and end frame number is floor of
-the number. Frame numbers start from 0.)code", false)
+      R"code(If the start/end timestamps are provided in file_list, you can interpret them
+as frame numbers instead of as timestamps.
+
+If floating point values have been provided, the start frame number is the highest possible number,
```

Comment:
This sentence also lost its meaning. It's not the same the floor and ceiling of a floating point number, which is rounding up or down respectively, than the "highest possible" or "lowest possible" number.
```suggestion
If floating point values have been provided, the start frame number will be rounded up and the end frame number will be rounded down.
```

Reply:
Done